app/views/form/check_answers.html.erb

```diff
@@ -19,11 +19,15 @@
     <% end %>
     <%= display_answered_questions_summary(subsection, @case_log) %>
 
-    <dl class="govuk-summary-list">
+    <%= govuk_summary_list do |summary_list| %>
```

Comment:
This shouldn't have an `=` should it?

Reply:
The initial wrapping function should, according to the docs: https://govuk-components.netlify.app/components/summary-list/#input-erb-a-summary-list-with-various-entries